### DISCUSSION TITLE
Is it possible to download multiple files by one click?

### DISCUSSION THREAD

**Discussion by fermelin** [selenide/selenide#2709] [Q&A] [answered]
I have a task to check that all files has been successfully downloaded after clicking on "download" button, but don't know how to do that. Is it possible in selenide?

**Comment by asolntsev** [comment, marked as answer]
@fermelin  No, Selenide doesn't have such a method.
But this is a good feature request.  ;)

Basically, we need to add method like `$.downloadFiles()` that returns `List<File>` (similar to method `$.download()` that returns a single `File`).
